src/stories/containers/RecognizedDelegates/KeyStats/KeyDescriptionCards/VisualizationCard/RelativeDelegateBar.tsx

```diff
@@ -0,0 +1,57 @@
+import styled from '@emotion/styled';
+import { useThemeContext } from '@ses/core/context/ThemeContext';
+import lightTheme from '@ses/styles/theme/light';
+import React from 'react';
+import type { WithIsLight } from '@ses/core/utils/typesHelpers';
+
+interface RelativeBarProps {
+  otherExpenses: number;
+  recognizedDelegates: number;
+}
+
+export const RelativeDelegateBar: React.FC<RelativeBarProps> = ({ otherExpenses, recognizedDelegates }) => {
+  const { isLight } = useThemeContext();
+
+  return (
+    <DelegateBar isLight={isLight}>
+      {otherExpenses > 0 && <OtherExpenses isLight={isLight} width={otherExpenses} />}
+      {recognizedDelegates > 0 && <RecognizedDelegates isLight={isLight} width={recognizedDelegates} />}
+    </DelegateBar>
+  );
+};
+
+const DelegateBar = styled.div<WithIsLight>(({ isLight }) => ({
+  position: 'relative',
+  width: '100%',
+  height: 24,
+  overflow: 'hidden',
+  borderRadius: 6,
+  background: isLight ? '#ECF1F3' : '#10191F',
+  alignItems: 'center',
+
+  [lightTheme.breakpoints.up('table_834')]: {
+    height: 12,
+  },
+}));
+
+const OtherExpenses = styled.div<WithIsLight & { width: number }>(({ isLight, width }) => ({
+  position: 'absolute',
+  top: 0,
+  right: 0,
+  background: isLight ? '#ECF1F3' : '#ECF1F3',
+
+  width: `${width}%`,
+  height: 24,
```

Comment:
the height here should be `100%` right? So you avoid to have issues with the parent because from 834 up, the height is 12 instead

Reply:
right , done